### PR TITLE
Docs: Removed line breaks from Python docs introduction.mdx

### PR DESCRIPTION
**Pull request by AdamGEmerson**
## What kind of change does this PR introduce?

Small fix to Python documentation where erroneous line breaks were showing in the introduction section.

## What is the current behavior?

Screenshot 2023-10-07 at 10 39 58 AM

## What is the new behavior?

Screenshot 2023-10-07 at 10 39 35 AM

## Additional context

As requested in PR #17676

### CODE REVIEW

**Comment by fsansalvadore** [comment]
Thanks @AdamGEmerson.
Can you please run `npm i && npm run format` in the root directory to fix Prettier?

**Review by Isaiah-Hamilton — APPROVED** [review]

**Comment by AdamGEmerson** [comment]
> Thanks @AdamGEmerson.
> Can you please run `npm i && npm run format` in the root directory to fix Prettier?

@fsansalvadore it appears that running prettier reintroduces the line breaks. 

I'm not sure why prettier is enforcing the line width on this section only. There are other lines in the same file that exceed the width and prettier does not touch them. Any ideas?

**Comment by Isaiah-Hamilton** [comment]
@AdamGEmerson you can use `<!-- prettier-ignore -->` to not format that line

**Comment by Isaiah-Hamilton** [comment]
or it might be for a range:
```mdx
<!-- prettier-ignore-start -->
<!-- SOMETHING AUTO-GENERATED BY TOOLS - START -->

| MY | AWESOME | AUTO-GENERATED | TABLE |
|-|-|-|-|
| a | b | c | d |

<!-- SOMETHING AUTO-GENERATED BY TOOLS - END -->
<!-- prettier-ignore-end -->
```

**Comment by AdamGEmerson** [comment]
I'll take a look, thanks.

Curious as to why the rest of the lines in the file aren't being effected even though they lack the ignore markup.

**Comment by AdamGEmerson** [comment]
So I know for future PRs, is there a reason that the checks are failing and do I need to take any additional action to run some of these checks? 

Thanks,

**Comment by AdamGEmerson** [comment]
@Isaiah-Hamilton Just following up on this.  Anything else I need to do to get this merged?  I am getting nightly emails that the vector embeddings for search have failed to generate but I don't know how to resolve.

**Comment by AdamGEmerson** [comment]
@fsansalvadore Any insight on getting this approved?  Vector embeddings continue to fail and I have looked at logs but can't make heads or tails of it.  I continue to get nightly emails about it.  If it is unimportant I can just close the PR.

Thanks,

**Comment by charislam** [comment]
Thanks for the PR, @AdamGEmerson! It was just missing some maintainer approvals to run mandatory checks, everything looks good and I just merged it!

**Comment by AdamGEmerson** [comment]
> Thanks for the PR, @AdamGEmerson! It was just missing some maintainer approvals to run mandatory checks, everything looks good and I just merged it!

Awesome!  Thank you!  Sorry for all the back and forth on such a small issue.

**Pull request merged**
